libraries/AP_HAL_ChibiOS/hwdef/crazyflie2/hwdef.dat

```diff
@@ -99,10 +99,10 @@ define STORAGE_FLASH_PAGE 1
 FLASH_RESERVE_START_KB 64
 
 # one IMU
-IMU Invensense I2C:0:0x69 ROTATION_YAW_180
```

Comment:
you should be able to leave both in, then we could support both old and new boards

Reply:
Ok. Added in latest commit